src/notebook/widget.ts

```diff
@@ -199,8 +199,6 @@ class NotebookWidget extends Widget {
       // widget so the lists stay in sync.
       widget = new BaseCellWidget(cell);
     }
-    widget.addClass(NB_CELL_CLASS);
-    widget.input.editor.addClass(NB_EDITOR_CLASS);
```

Comment:
I assume you removed these lines so that it would be easier to override this function, but still have all of the class logic work?